infra/nightly-resources/latex.js

```diff
@@ -0,0 +1,195 @@
+    rows = [
+      allRows.find((r) => r.spec_name === "bool"),
+      allRows.find((r) => r.spec_name === "bv4_base"),
+      allRows.find((r) => r.spec_name === "bv32"),
+      allRows.find((r) => r.spec_name === "rational_replicate"),
+    ];
+  } else if (tableName === "herbie") {
+    rows = allRows.filter((r) => r.spec_name != "rational_replicate");
+  } else {
+    rows = allRows;
+  }
+  let keys = {
+    domain: (row) => specNames[row.spec_name] || row.spec_name,
+    loc: (row) => row.loc,
+    renumo_time: (row) => `${row.rules.length} (${tryRound(row.time)})`,
+    baseline_time: (row) =>
+      `${
+        row.derivability.enumo_derives_baseline.lhs.can.length +
+        row.derivability.enumo_derives_baseline.lhs.cannot.length
+      } ${rulerTimes[row.spec_name] || ""}`,
+    enumo_derives_baseline: (row) => {
+      lhs_percentage = getDerivability(
+        row.derivability.enumo_derives_baseline.lhs
+      );
+      lhs_time = tryRound(row.derivability.enumo_derives_baseline.lhs?.time);
+      lhs_rhs_percentage = getDerivability(
+        row.derivability.enumo_derives_baseline.lhs_rhs
+      );
+      lhs_rhs_time = tryRound(
+        row.derivability.enumo_derives_baseline.lhs_rhs?.time
+      );
+      return `${lhs_percentage} (${lhs_time}), ${lhs_rhs_percentage} (${lhs_rhs_time})`;
+    },
+    baseline_derives_enumo: (row) => {
+      lhs_percentage = getDerivability(
+        row.derivability.baseline_derives_enumo.lhs
+      );
+      lhs_time = tryRound(row.derivability.baseline_derives_enumo.lhs?.time);
+      lhs_rhs_percentage = getDerivability(
+        row.derivability.baseline_derives_enumo.lhs_rhs
+      );
+      lhs_rhs_time = tryRound(
+        row.derivability.baseline_derives_enumo.lhs_rhs?.time
+      );
+      return `${lhs_percentage} ${
+        lhs_time ? `(${lhs_time})` : ""
+      }, ${lhs_rhs_percentage} ${lhs_rhs_time ? `(${lhs_rhs_time})` : ""}`;
+    },
+  };
+  let lines = [
+    String.raw`\begin{table}[h]`,
+    String.raw`\resizebox{\textwidth}{!}{%`,
+    String.raw`\begin{tabular}{lccccc}`,
+    String.raw`Domain   & \enumo LOC  & \# \enumo (Time)   & \# ${
+      macroNames[tableName]
+    } ${tableName === "ooplsa" ? "(Time)" : ""}  & \enumo $\rightarrow$ ${
+      macroNames[tableName]
+    } (Time) & ${
+      macroNames[tableName]
+    } $\rightarrow$ \enumo (Time)  \\ \cline{1-6}`,
+  ];
+  rows.forEach((row) => {
+    let latexRow = [];
+    Object.entries(keys).forEach(([key, f]) => {
+      latexRow.push(f(row));
+    });
+    lines.push(String.raw`${latexRow.join(" & ").replaceAll("%", "\\%")} \\`);
+  });
+
+  lines.push(String.raw`\end{tabular}%`);
+  lines.push(String.raw`}`);
+  lines = lines.concat(getCaption(tableName));
+  lines.push(String.raw`\label{table:${tableName}}`);
+  lines.push(String.raw`\end{table}`);
+
+  let elem = document.getElementById("latex");
+  elem.innerHTML = lines.join("\n");
+  elem.style.height = "200px";
+}
+
+function getCaption(version) {
```

Comment:
We should pass in the string from the outside instead of casing on the version here

Reply:
Sorry, what do you mean?

Reply:
Oh what I meant is
Do we do this check on the version anywhere else? It would be good to bundle all the specific stuff together in the code base